onelive/index.html

```diff
@@ -80,7 +80,7 @@ <h1 class="display-3 text-center mb-5">Take a look at our past OneLives</h1>
             </div>
             <div class="row justify-content-center mb-md-5">
                 <div class="col-9 text-center">
-                    <a href="archive/" class="btn btn-link border-0 btn-info">SEE MORE</a>
+                    <a href="past-programs/" class="btn btn-link border-0 btn-info">SEE MORE</a>
```

Comment:
I think we should keep the onelive archive as it is. Let's only change the scholarx archives.